seed/ts-sdk/imdb/no-custom-config/src/api/resources/imdb/client/Client.ts

```diff
@@ -43,10 +43,29 @@ export class Imdb {
      *         rating: 1.1
      *     })
      */
-    public async createMovie(
+    public createMovie(
         request: SeedApi.CreateMovieRequest,
         requestOptions?: Imdb.RequestOptions,
-    ): Promise<SeedApi.MovieId> {
+    ): core.HttpResponsePromise<SeedApi.MovieId> {
```

Comment:
Discussed offline - it's not ideal that we use the `core.*` import style here. We should follow-up by updating _all_ of the `core` import usages so that we can just reference this as `HttpResponsePromise` directly.